src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayController.cs

```diff
@@ -217,6 +217,13 @@ public async Task<IActionResult> EvaluateTradingNamePage(TradingNamePageViewMode
             return RedirectToAction("ViewApplication", new { viewModel.ApplicationId });
         }
 
+        #region 1.3 - OrganisationStatus
```

Comment:
If we're using regions, that's probably a sign that the controller class is getting too big and should be split up into separate classes

Reply:
Whether we are using #region or not it’s obvious that controller will grow in size and there is a need to refactor/organise it.

Reply:
No regions...EVER.